core/src/main/java/com/klaviyo/coresdk/networking/requests/TrackRequest.kt

```diff
@@ -32,11 +32,12 @@ internal class TrackRequest (
         }
      */
     override fun buildKlaviyoJsonQuery(): String {
+        addAnonymousIdToProps(customerProperties)
         val json = JSONObject(
             mapOf(
                 "token" to KlaviyoConfig.apiKey,
                 "event" to event,
-                "customer_properties" to JSONObject(customerProperties),
+                "customer_properties" to JSONObject(customerProperties.toMap()),
```

Comment:
Always wondered this: Do these properties get set on the customers why do we have two different sets of properties? I'm not necessarily expecting an answer to this just was curious.

Reply:
When it comes to track requests, `properties` is where you define properties about the event you are tracking, and `customer_properties` is where you define the properties that identity who triggered said event.

But in identify requests `properties` is where you define the properties about the person you are identifying, so the Klaviyo API is kind of misleading to work with

Reply:
Maybe that answers your question a bit?